### PR TITLE
Normalize RFCs

### PR DESCRIPTION
**Pull request by thomaseizinger**
Before implementing the changes from #23, I wanted to put all sentences on a new line for RFC1 to make the review easier. (This is the first commit in the PR.)

While doing that, I noticed that there is inconsistency in the way the ToC are presented so I normalized it according to the following rules:

1. "Table of Contents" by itself is not a header: This avoids having an entry in the ToC itself if generated by a tool.
2. The indentation is changed to 4 spaces to comply with the rest of the documents.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
```
Comment (D4nte) [inline .vscode/settings.json]: @thomaseizinger you are the only one using vscode for that. I am not sure it makes sense for this folder to be here.

File: RFC-008-ERC20.md [hunk 2]
```diff
@@ -5,16 +5,12 @@
...
 
-## Table of Contents
-<!-- markdown-toc start -->
+**Table of Contents**
 - [Description](#description)
-- [The ERC20 Asset](#the-erc20-asset-1)
+- [The ERC20 Asset](#the-erc20-asset)
     - [`quantity`](#quantity)
-    - [`address`](#address)
+    - [`token_contract`](#tokencontract)
```
Comment (D4nte) [inline RFC-008-ERC20.md]: this link does not work on github. I think it should be:
```suggestion
    - [`token_contract`](#token-contract)
```
or `#-token-contract-`

File: .vscode/settings.json (hunk 1 above)
Comment (thomaseizinger) [reply .vscode/settings.json]: We can check in the configuration of other editors as-well. I wouldn't object :)

File: RFC-008-ERC20.md (hunk 2 above)
Comment (thomaseizinger) [reply RFC-008-ERC20.md]: Will fix this one!

**Pull request merged**